nip07.ts

```diff
@@ -0,0 +1,16 @@
+import { EventTemplate, VerifiedEvent } from './core.ts'
+import { RelayRecord } from './index.ts'
+
+export interface Nip07 {
+  getPublicKey(): Promise<string>
+  signEvent(event: EventTemplate): Promise<VerifiedEvent>
```

Comment:
```suggestion
  signEvent(event: EventTemplate): Promise<Event>
```

An extension won't be able to add our special symbol.

Reply:
There is something that I have not understood, can you explain better this suggestion for me?
VerifiedEvent is exported and I use it and other special symbols like NostrEvent and EventTemplate in my apps, and I think that the extension will implement this interface using JavaScript and should return in the Promise a corresponding event with VerifiedEvent interface

Reply:
Look at the definition of VerifiedEvent. It has `[verifiedSymbol]: true`, which is something only nostr-tools can add to the object. An external extension can't do it.

Reply:
Understood, I changed it to ```NostrEvent```